assets/video_looper.ini

```diff
@@ -164,13 +164,10 @@ show_titles = false
 # Title duration in seconds. -1 means endless.
 title_duration = 10
 
-# Title font size, in 1/1000 screen height.
-title_size = 55
-
 # Any extra command line arguments to pass to omxplayer.  It is not recommended
 # that you change this unless you have a specific need to do so!  The audio and
 # video FIFO buffers are kept low to reduce clipping ends of movie at loop.
-extra_args = --no-osd --audio_fifo 0.01 --video_fifo 0.01
+extra_args = --no-osd --audio_fifo 0.01 --video_fifo 0.01 --align center
```

Comment:
maybe we should add --font-size 55 also (so that there is a default for the fontsize and people can figure it out more easily where to change the font size.